manage.py

```diff
@@ -24,20 +25,52 @@
 app = create_app()
 manager = Manager(app)
 
-migrate = Migrate(
-    app,
-    db,
-    directory=os.path.join(app.root_path, 'migrations')
-)
+MIGRATIONS_DIR = os.path.join(app.root_path, 'migrations')
+migrate = Migrate(app, db, directory=MIGRATIONS_DIR)
 manager.add_command('db', MigrateCommand)
 manager.add_command('runserver', ConfigServer(host='0.0.0.0', threaded=True))
 
 
+def _run_alembic_command(args):
+    """Helper to manage working directory and run given alembic commands"""
+    # Alembic looks for the alembic.ini file in CWD
+    # hop over there and then return to CWD
+    cwd = os.getcwd()
+    os.chdir(MIGRATIONS_DIR)
+
+    alembic.config.main(argv=args)
```

Comment:
Yes, I saw that, but it's a different path from being able to call with argv options - you'd need to do the equivalent of what's done in the main() therein, and then maintain it.  This hack looked like the path of least resistance.